<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QDialog" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>499</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonExitChat">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSendMessage">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDisconnect">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChoose">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose recipient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnectPorts">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect
 ports</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExitChat</sender>
   <signal>clicked()</signal>
   <receiver>Chat</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
